analyzer/restapi-plugin/README.md

```diff
@@ -0,0 +1,26 @@
+<p align="center">
+    <img src="https://user-images.githubusercontent.com/45048351/90059323-a7458d80-dceb-11ea-938b-a7288e784397.jpg">
+</p>
+<br/>
+<p align="center">
+    <a href="https://github.com/fasten-project/fasten/actions" alt="GitHub Workflow Status">
+        <img src="https://img.shields.io/github/workflow/status/fasten-project/fasten/Java%20CI?logo=GitHub%20Actions&logoColor=white&style=for-the-badge" /></a>
+    <!-- Here should be a link to Maven repo and version should be pulled from there. -->
+    <a href="https://github.com/fasten-project/fasten/" alt="GitHub Workflow Status">
+                <img src="https://img.shields.io/maven-central/v/fasten/restapi?label=version&logo=Apache%20Maven&style=for-the-badge" /></a>
+</p>
+<br/>
+
+Description goes here
```

Comment:
I have added a template README. The descriptions of the plugin needs to be added before merging this PR.